src/main/java/com/coveros/selenified/services/Response.java

```diff
@@ -249,14 +254,17 @@ public void assertContains(Map<String, Object> expectedPairs) {
      * @param expectedJson - the expected response json array
      */
     public void assertContains(JsonElement expectedJson) {
-        Success success = Success.FAIL;
+        boolean pass = false;
         if (array != null) {
-            success = array.contains(expectedJson) ? Success.PASS : Success.FAIL;
+            pass = array.contains(expectedJson) ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1125)